public/tools/randpass.go

```diff
@@ -0,0 +1,23 @@
+package tools
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+const (
+	passwordLength = 8
+	letters        = "abcdefghijklmnopqrstu@vwxyzABCDEFGHIJKL#MNOP*QRSTUVWXYZ0123456789"
```

Comment:
随机密码生成函数中的字符集缺少一些常用的特殊字符，如$、%、&、!等，这可能会降低密码的强度。

```suggestion
letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@#$%&*!"